packages/fx-core/tests/component/driver/botAadApp/create.test.ts

```diff
@@ -316,4 +316,88 @@ describe("botAadAppCreate", async () => {
       expect(e instanceof UnhandledError).to.be.true;
     }
   });
+
+  it("should output delete aad information when using microsoft tenant", async () => {
+    sinon
+      .stub(mockedDriverContext.m365TokenProvider, "getJsonObject")
+      .resolves(ok({ unique_name: "test@microsoft.com" }));
+    const args: any = {
+      name: expectedDisplayName,
+    };
+
+    sinon.stub(AadAppClient.prototype, "createAadApp").resolves({
+      id: expectedObjectId,
+      displayName: expectedDisplayName,
+      appId: expectedClientId,
+    } as AADApplication);
+
+    sinon.stub(AadAppClient.prototype, "generateClientSecret").resolves(expectedSecretText);
+
+    const result = await createBotAadAppDriver.handler(
+      args,
+      mockedDriverContext,
+      outputEnvVarNames
+    );
+
+    console.log(JSON.stringify(result));
+
+    expect(result.output.get(outputKeys.botId)).to.be.equal(expectedClientId);
+    expect(result.output.get(outputKeys.botPassword)).to.be.equal(expectedSecretText);
+  });
+
+  it("should not output delete aad information when using non microsoft tenant", async () => {
+    sinon
+      .stub(mockedDriverContext.m365TokenProvider, "getJsonObject")
+      .resolves(ok({ unique_name: "test@test.com" }));
+    const args: any = {
+      name: expectedDisplayName,
+    };
+
+    sinon.stub(AadAppClient.prototype, "createAadApp").resolves({
+      id: expectedObjectId,
+      displayName: expectedDisplayName,
+      appId: expectedClientId,
+    } as AADApplication);
+
+    sinon.stub(AadAppClient.prototype, "generateClientSecret").resolves(expectedSecretText);
+
+    const result = await createBotAadAppDriver.handler(
+      args,
+      mockedDriverContext,
+      outputEnvVarNames
+    );
+
+    console.log(JSON.stringify(result));
```

Comment:
Please also see whether we can remove the console.log here, as well as checking the summary string